### DISCUSSION TITLE
El SLA puede enviar un mensaje al segundo día y después de forma diaria?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#355] [2. Questions and Answers] [answered]
Hola

El caso que tengo es el siguiente: Quiero que el sistema me arroje un mensaje primero al segundo día y despues de forma diaria (despues del segundo día). Sería algo como, si me mandara el sistema un mensaje el jueves (partiendo desde hoy) y luego me enviara el mensaje para el viernes, sabado, domingo sucesivamente.

Yo lo tengo con 2 SLA de tipo static:
1.- Se envía solo cuando pasa al segundo día
2.- Se envía cuando pasa al tercer día y se enviara cada tres días.

No sé si el tipo dynamic funcionaría.

Gracias =>

**Comment by acedward** [comment, marked as answer]
No es posible, pero sería una buena funcionalidad tener **Frecuencia** + **Offset** de SLA  
así puedes decir Offset de 2 días, y con una frecuencia de 1 día para representar esto.

Alternativa 1
Primero crear un nuevo estado llamado ATRASADO.
Configurar un SLA de 2 dias, que al cumplirse envía la Alerta y mueve la tarea a ATRASADO.
En el estado ATRASADO tener un SLA de 1 día.

Alternativa 2
A través una funcionalidad personalizada programada en el FAAS, revisar períodicamente si es necesario enviar una alerta.

Alternativa 3
Implementar el Offset para SLA

**Reply by NicolasPLG** [reply]
hola @acedward muchas gracias por la información, revisaré ambas propuestas para ver cual más le acomoda al cliente.

Salu2
=>
